### ISSUE TITLE
Add support to remove Rank Math schemas

### ISSUE THREAD

**Issue by viralbaby** [Websitescanner/remove-schema#34]
Just deleted Rank Math last weekend, now it's throwing me old schema data all over the place -- including new posts.

Can't seem to clean everything up now that I'm using a new plugin and creating conflicts. Would be great if you could integrate support for Rank Math as well.

**Comment by timvaniersel** [comment]
@viralbaby do you have an example of a page with schema data that are leftovers from Rank Math? Which plugin are you using now?

**Comment by viralbaby** [comment]
Sure

Here is a post that I used SeoPRess to generate schema

https://search.google.com/test/rich-results?url=https://viral.baby/is-caulk-adhesive-strong-enough-to-glue-a-person-to-the-ceiling/

I wrote this post this weekend, after I deleted Rank Math

The post has 4 schemas - 1 review which is fine

RankMath still auto-generated 2 schemas -- 1 and 2

1 - article, generated automatically - didn't touch it
2 - videoobject, generated automatically
3 - videoobject - I created via SEOPRess and validates

Trying to figure out how to kill those 2 automatically created schemas

The one generated from SEOpress obviously is good    On Monday, November 16, 2020, 06:57:20 AM EST, Tim van Iersel <notifications@github.com> wrote:  
 
 


@viralbaby do you have an example of a page with schema data that are leftovers from Rank Math? Which plugin are you using now?

**Comment by timvaniersel** [comment]
Hi @viralbaby,

I've looked at the page and the schema isn't from RankMath. Probably generated by the theme or any other plugin. Unable to say based on the code which plugin generates the mentioned schema.

RankMath always adds the code `class="rank-math-schema"` to there script tag. Since websites with RM installed have the option to disable schema generated by RankMath from within the plugin settings I'll close this specific issue.

**Comment by viralbaby** [comment]
To my shock... you are absolutely correct...

It's my theme that's generating the previous schema and not Rank Math

Thanks for that.

Happy with SEOPRess so far, still playing with all the settings.

BTW and I'm sure that other have suggested it --- the ONLY reason that Rank Math got so big vs Yoast -- was producing a score /100 

Totally thin and cosmetic, but SEOPRess users existing and potential new customers would love it -- because we all like to get a number grade.

Cheers
Jeff    On Monday, November 16, 2020, 07:26:35 AM EST, Tim van Iersel <notifications@github.com> wrote:  
 
 


Hi @viralbaby,

I've looked at the page and the schema isn't from RankMath. Probably generated by the theme or any other plugin. Unable to say based on the code which plugin generates the mentioned schema.

RankMath always adds the code class="rank-math-schema" to there script tag. Since websites with RM installed have the option to disable schema generated by RankMath from within the plugin settings I'll close this specific issue.

**Comment by viralbaby** [comment]
quick problem -- see many of my old page titles in the browser are now %title% %sep% %sitename% in the browser bar -- I need to go back and change each one and manually add the text to correct

 https://viral.baby/inside-the-worlds-most-luxurious-secret-rooms/
Any quick fix?    On Monday, November 16, 2020, 01:34:24 PM EST, Jeff <webguru98@yahoo.com> wrote:  
 
  To my shock... you are absolutely correct...

It's my theme that's generating the previous schema and not Rank Math

Thanks for that.

Happy with SEOPRess so far, still playing with all the settings.

BTW and I'm sure that other have suggested it --- the ONLY reason that Rank Math got so big vs Yoast -- was producing a score /100 

Totally thin and cosmetic, but SEOPRess users existing and potential new customers would love it -- because we all like to get a number grade.

Cheers
Jeff    On Monday, November 16, 2020, 07:26:35 AM EST, Tim van Iersel <notifications@github.com> wrote:  
 
 


Hi @viralbaby,

I've looked at the page and the schema isn't from RankMath. Probably generated by the theme or any other plugin. Unable to say based on the code which plugin generates the mentioned schema.

RankMath always adds the code class="rank-math-schema" to there script tag. Since websites with RM installed have the option to disable schema generated by RankMath from within the plugin settings I'll close this specific issue.

**Comment by viralbaby** [comment]
Nevermind - looks like it was only 40-50 posts that I was able to manually fix

All good - no reply required
    On Monday, November 16, 2020, 02:42:29 PM EST, Jeff <webguru98@yahoo.com> wrote:  
 
  quick problem -- see many of my old page titles in the browser are now %title% %sep% %sitename% in the browser bar -- I need to go back and change each one and manually add the text to correct

 https://viral.baby/inside-the-worlds-most-luxurious-secret-rooms/
Any quick fix?    On Monday, November 16, 2020, 01:34:24 PM EST, Jeff <webguru98@yahoo.com> wrote:  
 
  To my shock... you are absolutely correct...

It's my theme that's generating the previous schema and not Rank Math

Thanks for that.

Happy with SEOPRess so far, still playing with all the settings.

BTW and I'm sure that other have suggested it --- the ONLY reason that Rank Math got so big vs Yoast -- was producing a score /100 

Totally thin and cosmetic, but SEOPRess users existing and potential new customers would love it -- because we all like to get a number grade.

Cheers
Jeff    On Monday, November 16, 2020, 07:26:35 AM EST, Tim van Iersel <notifications@github.com> wrote:  
 
 


Hi @viralbaby,

I've looked at the page and the schema isn't from RankMath. Probably generated by the theme or any other plugin. Unable to say based on the code which plugin generates the mentioned schema.

RankMath always adds the code class="rank-math-schema" to there script tag. Since websites with RM installed have the option to disable schema generated by RankMath from within the plugin settings I'll close this specific issue.

**Comment by viralbaby** [comment]
Tim

I'm getting a ton of 404 requests from the following 

seopress_404-seopress_404-asset-manifest-json-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2-2

Any ideas?    On Monday, November 16, 2020, 04:33:44 PM EST, Jeff <webguru98@yahoo.com> wrote:  
 
  Nevermind - looks like it was only 40-50 posts that I was able to manually fix

All good - no reply required
    On Monday, November 16, 2020, 02:42:29 PM EST, Jeff <webguru98@yahoo.com> wrote:  
 
  quick problem -- see many of my old page titles in the browser are now %title% %sep% %sitename% in the browser bar -- I need to go back and change each one and manually add the text to correct

 https://viral.baby/inside-the-worlds-most-luxurious-secret-rooms/
Any quick fix?    On Monday, November 16, 2020, 01:34:24 PM EST, Jeff <webguru98@yahoo.com> wrote:  
 
  To my shock... you are absolutely correct...

It's my theme that's generating the previous schema and not Rank Math

Thanks for that.

Happy with SEOPRess so far, still playing with all the settings.

BTW and I'm sure that other have suggested it --- the ONLY reason that Rank Math got so big vs Yoast -- was producing a score /100 

Totally thin and cosmetic, but SEOPRess users existing and potential new customers would love it -- because we all like to get a number grade.

Cheers
Jeff    On Monday, November 16, 2020, 07:26:35 AM EST, Tim van Iersel <notifications@github.com> wrote:  
 
 


Hi @viralbaby,

I've looked at the page and the schema isn't from RankMath. Probably generated by the theme or any other plugin. Unable to say based on the code which plugin generates the mentioned schema.

RankMath always adds the code class="rank-math-schema" to there script tag. Since websites with RM installed have the option to disable schema generated by RankMath from within the plugin settings I'll close this specific issue.
